### PR TITLE
Remove alchy and use SQLAlchemy instead

### LINKED ISSUE #57: Completely replace alchy with sqlservice
Make it work with this branch of chanjo ---> deprecate_alchy

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- This PR is testing also the interaction with [this chanjo PR](https://github.com/Clinical-Genomics/chanjo/pull/255) when it comes to creating a report
- Fix #57

**How to prepare for test**:
- ` pip install -e .`
- Start a local demo database --> `chanjo init --auto demodata`
- Load the transcripts --> `chanjo --config demodata/chanjo.yaml link demodata/hgnc.grch37p13.exons.bed`

Load 3 samples (coverage files are located in chanjo, not chanjo2)
- `chanjo --config demodata/chanjo.yaml load --sample sample1 --name sample1 --group-name test_group -g test_group <PATH_TO_CHANJO>/chanjo/init/demo-files/sample1.coverage.bed`
- `chanjo --config demodata/chanjo.yaml load --sample sample2 --name sample2 --group-name test_group -g test_group <PATH_TO_CHANJO>/chanjo/init/demo-files/sample2.coverage.bed`
- `chanjo --config demodata/chanjo.yaml load --sample sample3 --name sample3 --group-name test_group -g test_group <PATH_TO_CHANJO>/chanjo/init/demo-files/sample3.coverage.bed`

### How to test:
- Run the server --> `chanjo --config demodata/chanjo.yaml report --render html`
- Check the report in a brower at this address --> `http://localhost:5000/report?sample_id=sample1&sample_id=sample2&sample_id=sample3`
- Check the genes overview at `http://localhost:5000/genes?level=10&sample_id=sample1&sample_id=sample2&sample_id=sample3`
- Check the endpoint used by scout to create the MT coverage on the MT report:
```
curl --header "Content-Type: application/json" \
  --request POST \
  --data '{"chrom":"1","sample_ids":"sample1,sample2,sample3"}' \
http://localhost:5000/json_chrom_coverage
```

### Expected outcome:
- [ ] The report shoud be shown without errors

### Review:
- [ ] Code approved by DN
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
-alchy
 cairocffi
-chanjo>=4.1.0
+chanjo @ git+https://github.com/Clinical-Genomics/chanjo@deprecate_alchy
```
Comment (northwestwitch) [inline requirements.txt:2]: This won't work in PyPI, so we have to create a chanjo release first

**Comment by northwestwitch** [comment]
@dnil I've created a PR that works in a combo with a chanjo one

File: requirements.txt [hunk 2]
```diff
@@ -11,8 +11,6 @@ path.py
 pymysql
 pyscss
 six
-sqlservice
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply requirements.txt:15]: I gets installed by chanjo, so it's not that the lib isn't going to be there..

[the remark this answers is not in the document]
Comment (dnil) [reply requirements.txt:15]: If you actually import something from it, or use it directly, it should certainly be here. If you don't... 😜 
It's all good as it is now with this PR as well. Just saying. 😉

File: chanjo_report/server/extensions.py [hunk 3]
```diff
@@ -1,34 +1,4 @@
...
-
-        self.Query = session_options["query_cls"]
-
-    def __getattr__(self, attr):
-        """Delegate all other attributes to self.session"""
-        return getattr(self.session, attr)
-
-
-api = Alchy(Model=BASE)
+api = SQLAlchemy(model_class=BASE)
```
Comment (dnil) [inline chanjo_report/server/extensions.py:4]: 👍

**Review by dnil — APPROVED** [review]
This looks good to me, but we should test carefully with some real life cases in an integrated env (stage) before releasing!

**Comment by northwestwitch** [comment]
> This looks good to me, but we should test carefully with some real life cases in an integrated env (stage) before releasing!

I agree!

**Pull request merged**